translations/strings.yaml

```diff
@@ -64,21 +58,155 @@ settingsDev:
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
 
+serversView:
+  searchPlaceholder: Search countries
+  searchNoResultsLabel: No results. Try a different search
+
+# Changing very good to excellent within the in-client feedback form
+feedbackForm:
+  chooseCategory: Choose a category
+  excellentLabel: Excellent
+  # adding new content for the excellent choice in feedback form
+  selectionHeader: Give feedback
```

Comment:
It's awesome to have these strings, but they are not for 2.5. We have decided to implement this rating step in 2.6. Can we comment out to avoid extra work for the localizer? Just add '#' at the beginning of this line and the following 4.
In this way, when we are ready to implement this feature, we can remove your comments and the strings are already here.